packages/chart/src/components/Filters.jsx

```diff
@@ -38,6 +38,7 @@ const useFilters = () => {
     // reset to first item in values array.
     newFilters.map(filter => {
       filter.active = filter.values[0]
+      return null
```

Comment:
Did you test this? I know we've run into issues with `forEach` causing issues before with eslint but I wonder if we should be returning filter instead of null here.